sandbox-server/main.go

```diff
@@ -1,21 +1,28 @@
 package main
 
 import (
+	"bytes"
 	"context"
 	"fmt"
+	"io"
 	"log"
+	"mime/multipart"
 	"net/http"
 	"strings"
 	"time"
 	"unicode"
 
+	"archive/tar"
+
 	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/container"
 	"github.com/docker/docker/client"
 	gonanoid "github.com/matoous/go-nanoid/v2"
 )
 
-const dockerTimeout = time.Minute * 10
+const dockerTimeout = time.Minute * 1
+const firebaseStorageUrl = "https://firebasestorage.googleapis.com/v0/b/nutc-web.appspot.com/o"
+const firebaseApiKey = "AIzaSyCo2l3x2DMhg5CaNy1Pyvknk_GK8v34iUc"
```

Comment:
👀

Reply:
Didn't mean to commit the timeout change - woops. 

If the API key being here is a concern, it's already public in other parts of this repo.

Reply:
yeah the long term solution is move to a real storage provider, short term is prolly make the repo private until after the competition lol